### PR TITLE
Update Simplified & Traditional Chinese translations for upcoming 18.8

### PR DESCRIPTION
**Pull request by ASKLL-STAR**
## Checklist

- [x] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [x] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [x] Considered possible null pointers and out of bounds array indexing
- [x] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/simplified_chinese.txt [hunk 1]
```diff
@@ -1980,102 +1981,102 @@ Some fonts could not be loaded. Check the local console for details.
...
 
 [Direct touch input]
 Disabled
-== 
+== 禁用
 
 [Direct touch input]
 Active action
-== 
+== 激活的操作
```
Comment (Pioooooo) [inline data/languages/simplified_chinese.txt:2040]:
```suggestion
== 当前动作
```

File: data/languages/simplified_chinese.txt [hunk 2]
```diff
@@ -1980,102 +1981,102 @@ Some fonts could not be loaded. Check the local console for details.
...
-== 
+== 旁观时直接触摸操作
 
 Error loading touch controls
-== 
+== 加载触控设置错误
 
 Could not load touch controls from file. See local console for details.
-== 
+== 无法从文件加载触摸控件。检查本地控制台以获取详情。
```
Comment (Pioooooo) [inline data/languages/simplified_chinese.txt:2061]:
```suggestion
== 无法从文件加载触控设置。检查本地控制台以获取详情。
```

File: data/languages/simplified_chinese.txt [hunk 3]
```diff
@@ -1980,102 +1981,102 @@ Some fonts could not be loaded. Check the local console for details.
...
 
 Could not load default touch controls from file. See local console for details.
-== 
+== 无法从文件加载默认触摸控件。检查本地控制台以获取详情。
```
Comment (Pioooooo) [inline data/languages/simplified_chinese.txt:2064]:
```suggestion
== 无法从文件加载默认触控设置。检查本地控制台以获取详情。
```

File: data/languages/simplified_chinese.txt [hunk 4]
```diff
@@ -1980,102 +1981,102 @@ Some fonts could not be loaded. Check the local console for details.
...
 
 Could not load touch controls from clipboard. See local console for details.
-== 
+== 无法从剪贴板加载触控设置。检查本地控制台以获取详情
```
Comment (Pioooooo) [inline data/languages/simplified_chinese.txt:2067]:
```suggestion
== 无法从剪贴板加载触控设置。检查本地控制台以获取详情。
```

File: data/languages/simplified_chinese.txt [hunk 5]
```diff
@@ -1980,102 +1981,102 @@ Some fonts could not be loaded. Check the local console for details.
...
-== 
+== 瞄准
 
 Active: Fire
-== 
+== 激活：开火
 
 Active: Hook
-== 
+== 激活：钩索
```
Comment (Pioooooo) [inline data/languages/simplified_chinese.txt:2082]:
```suggestion
== 当前：钩索
```

**Review by Pioooooo — CHANGES_REQUESTED** [review]

File: data/languages/simplified_chinese.txt (hunk 5 above)
Comment (ASKLL-STAR) [reply data/languages/simplified_chinese.txt:2082]: 太好了是piooo，我这糟糕的翻译有救了

**Review by def- — APPROVED** [review]

**Pull request merged**